src/kirby/components/list/list.component.scss

```diff
@@ -2,11 +2,11 @@
 ul {
   list-style-type: none;
   padding-left: 0;
-  margin-top: 0;
+  margin: 0;
   font-size: font-size('s');
 
   li {
-    background-color: get-color('kirby-white');
+    border-bottom: 1px solid get-color('kirby-grey-2');
```

Comment:
Shouldn't this only be enabled when `showDivider=true`?

Reply:
True... left-over styling remains (that wasn't removed)... will do so!